Reimport css/css-overflow WPT tests

lukewarlow opened the pull request:
#### 03cda6345af7b15e1ed5fabe521fe0bbb459a271
<pre>
Reimport css/css-overflow WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=257204">https://bugs.webkit.org/show_bug.cgi?id=257204</a>

Reviewed by Tim Nguyen.

Reimport tests from the css/css-overflow folder. This is to get the latest scrollbar-gutter tests.

WPT Commit: 707500a0474977b3eea2873288d9c395cd123626

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/chrome-body-overflow-propagation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/incremental-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/margin-block-end-scroll-area-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/margin-block-end-scroll-area-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/orthogonal-flow-with-inline-end-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/orthogonal-flow-with-inline-end-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-007-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-008-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-009-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-cant-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-content-visual-overflow-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-invalidation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-border-radius-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-border-radius.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-inline-block-with-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-inline-block-with-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-no-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-no-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-padding-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-padding.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-recalc-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-resize-visibility-hidden-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/scrollbar-gutter-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-no-size-change.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-no-size-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-transform.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-transform.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-padding-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-padding.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-vertical-rl-dynamic-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-vertical-rl-dynamic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-zero-one-axis-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-zero-one-axis.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/table-header-group-overflow-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-editing-input-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-008-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-023.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-024-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-029-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-034.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-035-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-035.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-036-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-036.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-041-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-042-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-043-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-043.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-block-in-inline-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-block-in-inline-001.html: Added.

* LayoutTests/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/264477@main">https://commits.webkit.org/264477@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e25087de13557d4bfd47f4317de19c9d6a5e11a7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/7723 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/7994 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/8175 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/9364 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/7880 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/9984 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/7915 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/10749 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/7857 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/8974 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/7092 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/9472 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/6283 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/7031 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/14697 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/7437 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/7156 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/10564 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/7646 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/6232 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/6974 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/6941 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/1851 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/11185 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/7380 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rwlbuis reviewed (commented):
"this is to get" in the commit message. We also usually put the SHA the import is based on in the commit message.

nt1m reviewed (approved):
r=me if EWS is green

lukewarlow committed:
Reimport css/css-overflow WPT tests
https://bugs.webkit.org/show_bug.cgi?id=257204

Reviewed by Tim Nguyen.

Reimport tests from the css/css-overflow folder. This is to get the latest scrollbar-gutter tests.

WPT Commit: 707500a0474977b3eea2873288d9c395cd123626

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/chrome-body-overflow-propagation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/clip-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/display-flex-svg-overflow-default.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/incremental-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/margin-block-end-scroll-area-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/margin-block-end-scroll-area-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/orthogonal-flow-with-inline-end-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/orthogonal-flow-with-inline-end-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-auto-scrollbar-gutter-intrinsic-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-007-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-008-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-009-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-body-propagation-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-canvas.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-cant-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-content-visual-overflow-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-002-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-invalidation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-border-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-content-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-mul-column-padding-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value-with-border-radius.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-and-value.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-margin-visual-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-x-visible-y-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-clip-y-visible-x-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-border-radius-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-border-radius.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-display-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-object-position.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-non-replaced.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-scroll-replaced.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-img.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-inline-block-with-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-inline-block-with-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-no-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-no-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-overlay.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-padding-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-padding.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-recalc-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-big-border-small-content.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-intrinsic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-scroll-resize-visibility-hidden-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/overflow-video.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/paint-containment-svg.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/overflow-clip-margin.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/scrollbar-gutter-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-no-size-change.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-no-size-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-transform.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/resizer-transform.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-input-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-padding-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-padding.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-vertical-rl-dynamic-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-vertical-rl-dynamic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-zero-one-axis-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollable-overflow-zero-one-axis.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-abspos-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-dynamic-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-rtl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-lr-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-vertical-rl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/select-size-overflow-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/table-header-group-overflow-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/text-overflow-ellipsis-editing-input-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-008-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-023.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-024-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-029-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-034.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-035-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-035.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-036-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-036.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-041-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-042-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-043-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-043.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-block-in-inline-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/webkit-line-clamp-block-in-inline-001.html: Added.

* LayoutTests/TestExpectations:

Canonical link: https://commits.webkit.org/264477@main
